deployment/helm-k8s/templates/response/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.response.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "siembol.response.fullname" $ }}
+  namespace: {{ .Values.namespace }}
+spec:
+  replicas: {{ .Values.response.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ include "siembol.response.fullname" $ }}
+  strategy:
+    type: Recreate
```

Comment:
type: RollingUpdate + replicas should be 1 because siembol response is processing alerts in kafka streams and we do no want to have multiple instances in the same time